website/blog/2020-07-06-version-0.63.md

```diff
@@ -0,0 +1,153 @@
+
+These components make it easy to get started making things interactive. However, because they include built-in styles and effects that don’t match the system interaction, they make it easy to tell when experiences are written with React Native. Further, as React Native has grown and our bar for high-quality applications has gone up, these components haven't grown with it. React Native now supports platforms like Web, Desktop, and TV, but support for additional input modalities has been lacking. React Native needs something that meets users' expectations and gives them the best possible interaction experience on all platforms.
+
+To address these problems, we are shipping a new core component called Pressable. This component can be used to detect various types of interactions. It can be extended by the platform to include things like hover, blur, focus, and more. We expect that most people will build and share components utilizing Pressable under the hood instead of relying on the default experience of something like TouchableOpacity.
+
+```jsx
+import { Pressable, Text } from 'react-native';
+
+<Pressable
+  onPress={() => {
+    console.log('pressed');
+  }}
+  style={({ pressed }) => ({
+    backgroundColor: pressed ? 'lightskyblue' : 'white'
+  })}>
+  <Text style={styles.text}>Press Me!</Text>
+</Pressable>;
+```
+
+<p style="text-align: center; font-size: 12px">
+A simple example of a Pressable component in action
+</p>
+
+You can learn more about it from [the documentation](https://reactnative.dev/docs/next/pressable).
+
+## Native Colors (PlatformColor, DynamicColorIOS)
+
+Every native platform has the concept of system-defined colors. Colors that automatically respond to system theme settings such as Light or Dark mode, accessibility settings such as a High Contrast mode, and even its context within the app such as the traits of a containing view or window.
+
+While it is possible to detect some of these settings via the [`Appearance`](https://reactnative.dev/docs/appearance#getcolorscheme) API and/or [`AccessibilityInfo`](https://reactnative.dev/docs/accessibilityinfo#isgrayscaleenabled) and set your styles accordingly, such abstractions are not only costly to develop but are just approximating the appearance of native colors. These inconsistencies are particularly noticeable when working on a hybrid application, where React Native elements co-exist next to the native ones.
+
+React Native now provides an out-of-the-box solution to use these system colors. `PlatformColor()` is a new API that can be used like any other color in React Native.
+
+For example, on iOS, the [system provides a color called `labelColor`](https://developer.apple.com/documentation/uikit/uicolor/ui_element_colors?language=objc). That can be used in React Native with `PlatformColor` like this:
+
+```jsx
+import { Text, PlatformColor } from 'react-native';
+
+<Text style={{color: PlatformColor(‘labelColor’) }}>
```

Comment:
The decorated apostrophes has been used here instead of single quote character (this causes syntax error).

Reply:
I need to be more careful while moving from Google Docs next time.